matrix-neoboard-widget/package.json

```diff
@@ -23,7 +23,8 @@
     "react-redux": "^9.1.2"
   },
   "resolutions": {
-    "fork-ts-checker-webpack-plugin": "^6.5.3"
+    "fork-ts-checker-webpack-plugin": "^6.5.3",
+    "@vitejs/plugin-react-swc": "git+https://github.com/vitejs/vite-plugin-react-swc.git"
```

Comment:
This is required as we require https://github.com/vitejs/vite-plugin-react-swc/commit/beb09db39d4930a0013916508648e21d1d76d3e9